app/assets/javascripts/modules/track-click.js

```diff
@@ -5,24 +5,36 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
 
   Modules.TrackClick = function () {
     this.start = function (element) {
-      element.on('click', trackClick);
+      var trackable = '[data-track-category][data-track-action]';
 
-      var options = {transport: 'beacon'},
-          category = element.attr('data-track-category'),
-          action = element.attr('data-track-action'),
-          label = element.attr('data-track-label'),
-          dimension = element.attr('data-track-dimension'),
-          dimensionIndex = element.attr('data-track-dimension-index');
-
-      if (label) {
-        options.label = label;
+      if (element.is(trackable)) {
+        element.on('click', trackClick);
+      } else {
+        element.on('click', trackable, trackClick);
       }
```

Comment:
I'd agree if this wasn't generic, but since this could be used anywhere and with any amount of links, I'd rather us just do this now since memory leaks are hard to debug and crop up at the worst times.